plugin/package.json

```diff
@@ -2,8 +2,11 @@
   "devDependencies": {
     "@react-native/eslint-config": "^0.72.1",
     "@types/node": "^18.15.11",
+    "@typescript-eslint/eslint-plugin": "^7.0.2",
+    "@typescript-eslint/parser": "^7.0.2",
     "esbuild": "^0.17.11",
-    "eslint": "^8.35.0",
+    "eslint": "^8.56.0",
```

Comment:
Could you update all eslint dependencies in each of our `package.json`s? So we have the same version in all the examples:
<img width="259" alt="image" src="https://github.com/software-mansion/react-native-reanimated/assets/56199675/fdcfdc86-1646-4d4f-99cf-fb6f407f5b9a">

Also `@typescript-eslint/eslint-plugin` in main folder has version `"^6.19.0"`, could you make sure all the versions match?

Probably this change should be done in separate PR preceding this one

Reply:
We should probably use ^8.0.0-0 everywhere for simplicity.